github.js

```diff
@@ -91,7 +92,8 @@ var GithubLocation = function(options, ui) {
     cwd: options.tmpDir,
     timeout: options.timeout * 1000,
     killSignal: 'SIGKILL',
-    maxBuffer: 2 * 1024 * 1024
+    maxBuffer: 2 * 1024 * 1024,
+    "strict-ssl": this.strictSSL
```

Comment:
This options are specifically for _child_process_ execution options in Node. I don't think `strict-ssl` is an option here?